### DISCUSSION TITLE
How to set MadMax Number of buckets for phase 3+4

### DISCUSSION THREAD

**Discussion by ray-su** [ericaltendorf/plotman#859] [Q&A]
Hi, how do I set number of buckets for phase 3+4 for MadMax?

from MadMax doc:
 -u, --buckets arg    Number of buckets (default = 256)
  -v, --buckets3 arg   Number of buckets for phase 3+4 (default = buckets)

**Comment by altendky** [comment]
That isn't presently possible.  We need to review all the options and get them all available in plotman.
